examples/predictors/lstm/xprize_predictor.py

```diff
@@ -2,6 +2,9 @@
 
 import os
 
+# Suppress noisy Tensorflow debug logging
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
```

Comment:
This is a bit drastic as it silences all the messages from Tensorflow, but it's not like anyone ever reads them. One of the more annoying ones is due to the lack of CUDA, even on non-GPU-enabled hardware. I learned there is a "lean" Tensorflow package, `tensorflow-cpu`, that doesn't include support for GPUs, which we could look into using some day.

Reply:
ok